r/R/dplyr-funcs-type.R

```diff
@@ -76,6 +76,53 @@ register_bindings_type_cast <- function() {
   register_binding("as.numeric", function(x) {
     Expression$create("cast", x, options = cast_options(to_type = float64()))
   })
+  register_binding("as.Date", function(x,
+                                       format = NULL,
+                                       origin = "1970-01-01",
+                                       tz = "UTC") {
+    # base::as.Date() first converts to UTC and then extracts the date, which is
+    # why we need to go through timestamp() first - see unit tests for the real
+    # life impact of the difference between lubridate::date() and base::as.Date()
+    # browser()
```

Comment:
I know you'll take care of these, but wanted to flag them since they don't pop up too noisily

Reply:
Done in a prev commit.